src/surfing_penguin/models.py

```diff
@@ -28,3 +29,26 @@ def __repr__(self):
 
     def check_password(self, password):
         return check_password_hash(self.password_hash, password)
+
+
+class Survey(Base):
+    __tablename__ = 'survey'
+    id = Column(Integer, primary_key=True)
+    surveyname = Column(String(128))
+    # TODO: add author
+    # TODO: add bidirectional relastionship with question
+    questions = relationship("Question")
+    # TODO: add answer list
+
+    def __init__(self, name):
+        self.surveyname = name
+
+
+class Question(Base):
+    # TODO: add detail in what should in a question
+    __tablename__ = 'question'
+    id = Column(Integer, primary_key=True)
+    title = Column(String(128))
+    content = Column(String(1024))
+    survey_id = Column(Integer, ForeignKey('survey.id'))
+    # survey = relationship("Survey", back_populates="question")
```

Comment:
Do not push comment unless it will be uncommented under some circumstances. If that is the case, you have to explain ho to use it.